freeipa/src/main/resources/freeipa-salt/salt/freeipa/backups.sls

```diff
@@ -15,6 +15,11 @@
     - mode: 640
 
 {% if salt['pillar.get']('freeipa:backup:monthly_full_enabled') %}
+freeipa_backoff_monthly_full:
+  cmd.run:
+    - name: /sbin/anacron -u cron.monthly
+    - onlyif: test ! -s /var/spool/anacron/cron.monthly
```

Comment:
Is it possible that the anacron monthly backup runs once before the FreeIPA backup is added to it? If so, then this file would exist but the backup would not.

Reply:
initial backup is a different state. also `-u` is for skipping the actual run

Reply:
no the anacron is not going to the cron.monthly until at least 45 minutes after the initial instance creation and then have an up to 45 minute random delay added onto that.  Unless the instance creation is taking longer than that to reach that point it will be fine.  We can force this to not run by disabling anacron completely during the salt process and explicitly turn it on at the end of the salt run.  We can do this by putting 0anacron in the /etc/cron.hourly/jobs.deny file until we want to explicitly remove it.

Reply:
And actually on thinking about this.  In this case if the cron.monthly does get run prior to this that's fine, means it wont run the monthly till next month which is what we want anyways.